src/cm_device.cpp

```diff
@@ -1581,6 +1581,46 @@ INT CmDevice::GetGenStepInfo(UINT platform, char *&stepinfostr)
 	return CM_SUCCESS;
 }
 
+CM_RT_API INT CmDevice::SetSuggestedL3Config( L3_SUGGEST_CONFIG l3_s_c)
+{
+    INT upper_bound;
+    const L3_CONFIG_REGISTER_VALUES *l3_c;
+
+    switch (m_Platform)
+    {
+        case IGFX_GEN6_CORE:
+            return CM_FAILURE;
+
+        case IGFX_GEN7_CORE:
+            upper_bound = HSW_L3_CONFIG_NUM;
+            l3_c = &IVB_L3_PLANE[0];
+            break;
+
+        case IGFX_GEN7_5_CORE:
+            upper_bound = HSW_L3_CONFIG_NUM;
+            l3_c = HSW_L3_PLANE;
+            break;
+
+        case IGFX_GEN8_CORE:
+            upper_bound = BDW_L3_CONFIG_NUM;
+            l3_c = BDW_L3_PLANE;
+            break;
+
+         default:
```

Comment:
skylake/Gen8 was already supported (pushed by Lina) , please add L3# configuration for this platform.